src/components/Category.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState } from 'react';
+import styled, { keyframes } from 'styled-components';
+
+import { CategoryFetchers } from '../fetchers/categories';
+
+import { useHistory } from 'react-router-dom';
+
+import { ReactComponent as OpenCloseIcon } from '../assets/icon/category/category_open_close_icon.svg';
+
+interface CategoryProps {
+  id: number;
+  title: string;
+}
+
+export default function Category(props: CategoryProps) {
+  const { id, title } = props;
+  const [childCategories, setChildCategories] = useState([]);
+  const [isSelected, setIsSelected] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      setChildCategories(
+        await CategoryFetchers.getChilds({
+          parent_id: id,
+          type: 'child',
+        })
+      );
+    })();
+  }, []);
+  async function handleCategoryClicked() {
+    setIsSelected(!isSelected);
+  }
```

Comment:
여기서는 화살표함수 안쓰고  function 키워드 사용해서 만든 이유가 있을까요?? 화살표함수쓰는게 조을거같아용
그리구 async 붙일 이유가 있을까요?/?
그리고 `useState`의 setState함수이용할때 이전state기반으로 업데이트 할 수 있도록 명확하게 쓰는게 더 안전할 것 같아요

```suggestion
const handleCategoryClicked=async()=> {
    setIsSelected((isSelected)=>!isSelected);
  }
```